src/store/ChartTitleStore.js

```diff
@@ -39,52 +38,32 @@ export default class ChartTitleStore {
     }
 
     onContextReady = () => {
-        this.context.stx.append('createDataSet', this.update.bind(this));
+        this.mainStore.chart.feed.onMasterDataUpdate(this.update.bind(this));
         this.update();
     };
 
-    update() {
+    @action.bound async update(quote) {
```

Comment:
why is this marked as `async` ?

Reply:
ops. Was testing out if it would make it faster. It doesn't. Lemme remove it.